src/index.js

```diff
@@ -1,16 +1,56 @@
 /* eslint-disable linebreak-style */
 
-import _ from 'lodash';
+// import _ from 'lodash';
 import './style.css';
 
-function component() {
-  const element = document.createElement('div');
+const taskList = [
+  {
+    description: 'Wake up',
+    completed: true,
+    index: 0,
+  },
+  {
+    description: 'Have breakfast',
+    completed: true,
+    index: 1,
+  },
+  {
+    description: 'Get ready',
+    completed: true,
+    index: 2,
+  },
+  {
+    description: 'Start working',
+    completed: true,
+    index: 3,
+  },
+];
 
-  // Lodash, now imported by this script
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-  element.classList.add('hello');
+const displayTasks = () => {
+  const listSection = document.querySelector('.tasks_list');
+  taskList.forEach((e) => {
+    if (e.index === 0) {
+      listSection.innerHTML += `
+    <li class="add_task">
+      <p>Add to your list...</p>
```

Comment:
[ Optional Changes ]

I'm marking this change as optional because it's not a major change and you did well in structuring your to-do list app. I highly recommend you consider fixing this change as it'll help you on your next milestone.

- [ ] Currently, you're having a static text in a `p` tag which looks as if it's an input field just like the original design, but technically, it's not supposed to be a `p` tag, it's an input field that'll capture a todo one may enter and it gets stored to localStorage. Kindly change this `p` tag to an input field like this `<input type="text" placeholder="Add to your list" />` and then style it and remove the border. 👍🏽